ie/apply-action.go

```diff
@@ -67,26 +108,20 @@ func (i *IE) HasFORW() bool {
 	if err != nil {
 		return false
 	}
-	if len(v) < 1 {
-		return false
-	}
 	return has2ndBit(v[0])
 }
 
 // HasBUFF reports whether an IE has BUFF bit.
 func (i *IE) HasBUFF() bool {
-	if len(i.Payload) < 1 {
-		return false
+	v, err := i.ApplyAction()
```

Comment:
How about rewrite as below:
```
	switch i.Type {
	case CreateFAR, UpdateFAR, ApplyAction:
		v, err := i.ApplyAction()
		if err != nil {
			return false
		}
		return has3rdBit(v[0])
	case DownlinkDataReport, DataStatus:
		v, err := i.DataStatus()
		if err != nil {
			return false
		}
		return has2ndBit(v)
	default:
		return false
	}
```

Reply:
I removed the related commit from this branch.